pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "freebox-api"
 version = "1.1.0"
 description = "Provides asynchronous authentication and access to Freebox servers"
 authors = ["stilllman <luc_touraille@yahoo.fr>", "quentame <polletquentin74@me.com>", "HACF <contact@hacf.fr>"]
-license = "GNU GPL v3"
+license = "GPL-3.0-or-later"
```

Comment:
Changed it to `GPL-3.0-or-later` based on the `LICENSE.rst` file.
https://github.com/hacf-fr/freebox-api/blob/8d2de4073fbe766c864917529fab131a180b6dcd/LICENSE.rst?plain=1#L581-L584